src/System.Data.SQLite/NativeMethods.cs

```diff
@@ -97,6 +97,9 @@ internal static class NativeMethods
 		[DllImport(c_dllName, CallingConvention = c_callingConvention)]
 		public static extern IntPtr sqlite3_errstr(SQLiteErrorCode rc);
 
+		[DllImport(c_dllName, CallingConvention = c_callingConvention)]
+		public unsafe static extern int sqlite3_file_control(SqliteDatabaseHandle db, [MarshalAs(UnmanagedType.LPStr)] string zDbName, int op, int * userData);
```

Comment:
`unsafe` code and `int *` is not ideal, but I don't see a better alternative for this API.

Reply:
Yeah, I would have tried harder to avoid it if we didn't already use `unsafe` in this library.